main.cpp

```diff
@@ -20,67 +20,96 @@ You are going to write 10 UDTs in project3.
-    Name 3 things you'll find on the:    Infotainment System
-        1) Radio
-        2) Music Streaming system
-        3) Navigation system 
-    Name 3 things you can do with the:   Infotainment System
-        1) Listen to satellite radio program
-        2) listen to music streamed from a portable device
-        3) Get directions 
-
-Sub Object 5: Seat 
-    Name 3 things you'll find on the:    Seat
-        1) protection
-        2) tilt controls
-        3) height control
-    Name 2 things you can do with the:   Seat
-        1) sit on it
-        2) adjust its height
+1)  Fill in #1 - 4 with a random UDT in plain english
+    These 4 UDTs do not need to be related.
+        a) For each plain-english UDT, write out 5 traits or properties and 3 things it can do
+        b) remember: these properties will eventually become primitives
 */
+
+/*
+ example:  
+ 
+Thing: Car Wash   
+    5 properties:
+        - number of vacuum cleaners
+        - number of eco-friendly cleaning supplies
+        - stores the amount of water used per week.
+        - stores amount of profit made per week
+        - number of cars serviced per day
+    3 things it can do:
+        - wash and wax car
+        - charge customer
+        - detail the car interior
+
+    Notice that I did not use "has a vacuum cleaner" or "Has eco-friendly cleaning supplies" as one of the properties
+    
+    Writing 'has a ___" checks whether or not your object **has the ability to do something**.
+    Instead, I wrote "number of vacuum cleaners" and "number of eco-friendly cleaning supplies".  
+    
+    These are specific objects or amounts. 
+    
+    In C++ terms, this means to I want you to avoid using 'bool' (has a) as a member variable type.
+    Instead, prefer the other primitive types.
+ */
+
+/*
+Thing 1) Orchestral score
+5 properties:
+    1) number of instruments
+    2) displays notes for each instruments
+    3) instrument list
+    4) contains rehearsal marks
+    5) contains tempo markings
+3 things it can do:
+    1) provides roadmap for the conductor
+    2) gives information for the copyist to generate parts
+    3) provides information for hiring musicians
+ */
+
+/*
+Thing 2) Calendar
+5 properties:
+    1) page size
+    2) paper type
+    3) binder type
+    4) contains year information
+    5) displays national holliday dates
+3 things it can do:
+    1) organize person's schedule
+    2) block time off
+    3) remind important events
+ */
+
+/*
+Thing 3) Bookshelf
+5 properties:
+    1) dimensions
+    2) number of doors
+    3) color
+    4) material 
+    5) number of shelves
+3 things it can do:
+    1) stores books
+    2) displays objects
+    3) hide wall defects
+ */
+
+/*
+Thing 4) Road
+5 properties:
+    1) number of lanes
+    2) allows a certain number of vehicles to pass through per hour
+    3) has a starting and ending location
+    4) number of toll sections
+    5) requires a yearly maintenance cost
```

Comment:
Some of these don't translate to properties that can be represented as primitives.  They should be rephrased.
for example: `5) requires a yearly maintenance cost`.   Can you name a road that does **not** require a yearly maintenance cost?   This plain-english sentence comes across like you intend to use a boolean to represent whether or not the road has a maintenance cost.   a better choice would be: `yearly maintenance cost in dollars`
- [ ] revise